src/docs/product/alerts/index.mdx

```diff
@@ -54,6 +54,14 @@ By default all users can mute alerts for everyone. Sentry users with manager or
 
 Issue alerts can be muted on the alert rule details page by clicking the "Mute" button. This will mute the alert from notifying you until you click "Unmute". If you want to mute the alert from firing entirely, select "Mute for everyone" from the dropdown. If the only alert action is set to notify an integration, the only option available will be "Mute for everyone". If there are multiple alert actions which notify an integration and notify a user or team and you select "Mute for me", the integration alert will still fire.
 
+<Note>
+
+Muting metric alerts is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+Metric alerts can also be muted in the same way as issue alerts. Click [here](/product/alerts/alert-types/) to learn about the differences between issue alert and metric alerts.
+
```

Comment:
```suggestion


```
Thanks for bringing this up to my attention @ceorourke! I've simplified the page a bit more by adding the info into one paragraph at the top. But in general, we're going to re-look at the way this page and its subpages are structured. It seems really repetitive to talk about the different alert types on the index page and then also have a dedicated page to alert types ;p